doc/nrf/gs_dtm_testing.rst

```diff
@@ -0,0 +1,118 @@
+.. _gs_dtm_testing:
+
+Direct Test Mode terminal connection
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Follow the instructions in this section to connect with the DK which is running the DTM to send commands and receive responses.
+
+The Bluetooth Low Energy DTM UART interface standard specifies the following configuration:
+
+* Eight data bits
+* No parity
+* One stop bit
+* No hardware flow control
+* A selection of bit rates from 9600 to 1000000, one of which must be supported by the DUT.
+  It might be possible to run other bit rates by experimenting with parameters.
+
+.. note::
+   The default bit rate of the DTM UART driver is 19200 bps, which is supported by most certified testers.
+
+You have to send all commands as a two-byte Hex number. The responses must have the same format.
+
+Connect with RealTerm (Windows)
+*******************************
+
+The RealTerm start window:
```

Comment:
You might want to use this text as a caption text under the image instead of having it before the image.